idd/Energy+.idd.in

```diff
@@ -63383,12 +63383,15 @@ Boiler:Steam,
        \key OtherFuel1
        \key OtherFuel2
   N1 , \field Maximum Operating Pressure
-       \units Kpa
+       \required-field
+       \units Pa
   N2 , \field Theoretical Efficiency
+       \required-field
```

Comment:
Same comment here.  Once we make this a required field in 8.8, without a default, we could break user files where they had it blank in 8.7. (Would it run blank in 8.7?)  Anyway, just need to resolve here before this goes in.